src/analysis/p_series.lean

```diff
@@ -229,3 +229,62 @@ by simp [← nnreal.summable_coe]
 
 lemma nnreal.summable_one_div_rpow {p : ℝ} : summable (λ n, 1 / n ^ p : ℕ → ℝ≥0) ↔ 1 < p :=
 by simp
+
+section
+
+open finset
+
+variables {α : Type*} [linear_ordered_field α]
+
+lemma sum_Ioc_div_sq_le_sub {k n : ℕ} (hk : k ≠ 0) (h : k ≤ n) :
+  ∑ i in Ioc k n, (1 : α) / i ^ 2 ≤ 1 / k - 1 / n :=
```

Comment:
I think `1 / x` is not simp-normal-form. It's probably worth it to have the LHS in SNF.